client/project.go

```diff
@@ -22,14 +22,14 @@ func (self *ApiClient) Project(id string) (Project, error) {
 	return result, nil
 }
 
-func (self *ApiClient) ProjectCreate(name string) (Project, error) {
+func (self *ApiClient) ProjectCreate(name string, description string) (Project, error) {
```

Comment:
i think we should pass an options object that contains the fields, as we did in other places

Reply:
Feels a little out of scope for this PR.
I'll open an issue for that as it's also relevant for the configuration variable parameters.